TestClasses/TPCTest.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python3
+import sys
+import time
+import numpy as np
+from BaseTest import BaseTest
+import yaml
+import random
+import subprocess
+import os
+
+class TPCTest(BaseTest):
+    def __init__(self, configFile):
+        super().__init__()
+        self.srcServer = None
+        self.destinServer = None
+        self.siteA = None #{'CEPH-SVC16':['ceph-svc16.gridpp.rl.ac.uk', 'dteam:/test/tst.txt']} # Defaults if no config file? 
+        self.siteB = None #{'GOLIAS100': ['golias100.farm.particle.cz', 'dpm/farm.particle.cz/home/dteam/test/tst.txt']}
+        self.checksums = []
+        self.basecmd = {'copy'      : {'xrd' : ['xrdcp', 'args', '--tpc', 'delegate', 'only', 'siteA', 'src', 'siteB', 'dest'], 
+                                       'gfal': ['gfal-copy', 'siteA', 'src', 'siteB', 'dest', 'args']}, 
+                        'checksum'  : {'xrd' : ['xrdfs', 'siteB', 'args', 'dest'], 
+                                       'gfal': ['gfal-sum', 'siteB', 'dest', 'ADLER32']},  
+                        'delete'    : {'xrd' : ['xrdfs', 'siteB', 'args', 'dest'], 
+                                       'gfal': ['gfal-rm', 'siteB', 'dest']},          
+                        'load'      : {'xrd' : ['xrdcp','siteA', 'src', 'siteB', 'dest', '-d1', '--force'], 
```

Comment:
is 'load' testing the testing done with redirectors? if so the --tpc delegate only should also be put here